hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/connector/map/LazyDefiningSpecificMemberPms.java

```diff
@@ -113,11 +97,26 @@ public void init(@Nonnull Context context) throws Exception {
             }
```

Comment:
is `partitionAssignment` value sorted? if so, maybe better to use binary search? In default 3-member cluster there will be 90 partitions per member and on average we need to check half of members.

Reply:
BTW, you do that in test

Reply:
Its a good catch, thanks.